test/specs/repo/property.ts

```diff
@@ -58,18 +57,18 @@ describe( 'Property', () => {
 			} );
 
 			it( 'Should be able to add reference to property', async () => {
-				await Property.addReference.click();
+				await Property.addReferenceLink.click();
 				// fill out property id for reference
 				await $( '.ui-entityselector-input' ).isFocused();
 				await browser.keys( stringPropertyId.split( '' ) );
 				await $( propertyIdSelector( stringPropertyId ) ).click();
 				await browser.keys( 'REFERENCE'.split( '' ) );
-				await Property.saveStatement.click();
+				await Property.saveStatementLink.click();
 			} );
 
 			it( 'Should be able to see added reference', async () => {
 				await $( '=1 reference' ).click();
-				await $( '=REFERENCE' );
+				await $( '=REFERENCE2' );
```

Comment:
Where does this 2 come from?

Reply:
It is actually a typo, I'm certain of it. What bothers me is that the test seemed to pass!